### ISSUE TITLE
[Documentation] Alternative/Vanilla methods of using the Archives

### ISSUE THREAD

**Issue by KadTheHunter** [KadTheHunter/ShulkerArchives#102]
As demonstrated by CaliSakuraium, methods such as `/data merge` are not easily useable with the provided SNBT.

I am unsure how much/what can be done to address this, but it's worth looking into.

Secondary part of this issue/Reminder for me:

Draft an update to tweak existing UsingNBT info, and complete the remaining two sections.

**Comment by KadTheHunter** [comment]
After extensive research, I have decided not to support the `/data merge` command in any way whatsoever..

Aside from it being annoying as hell to use with *any* of the SNBT here due to the length of said SNBT, I encountered a number of oddities:

- When setting the coordinates, it refused to acknowledge there was a chest (or any storage block, or any block entity) at the given coordinates, but would happily acknowledge that chest when the Y coordinate was lowered by 1.
- Things such as (BlockEntityTag:) make it not work at all; It will *say* it modified the data, but it won't do shit. This seems like a simple fix, just remove that bit, but in removing it you also have to remove the display:{) tag, which means the kit will have no name or lore at all. 
- Either I'm retarded, or there's no obvious way to work with individual items, only Block Entities such as chests/shulkers. This makes 2 of the 3 archives inaccessible.

TL;DR ***Fuck `/data merge`, and the peasants who use it. Get a real NBT Editor, and inb4 "mUh VaNiLlA cLienT", if you're dumb enough to be using a vanilla client for anything involving NBT or Creative Mode, you don't need to be messing with NBT. Like, at all.***
